source/mapping-rules/new-rules-from-relational.txt

```diff
@@ -46,7 +46,35 @@ Steps
 
 #. Define the name of the target MongoDB collection.
 
-#. Define the options for the new mapping rule.
+#. Define the options for the new mapping rule, when defining the 
```

Comment:
I couldnt find a way to include this content as an external file without resetting the list numbers